tutorecommerce/plugin.py

```diff
@@ -78,6 +82,27 @@
     },
 }
 
+
+CORE_MFE_APPS: dict[str, MFE_ATTRS_TYPE] = {
+    "orders": {
+        "repository": "https://github.com/edx/frontend-app-ecommerce",
+        "port": 1996,
+    },
+    "payment": {
+        "repository": "https://github.com/edx/frontend-app-payment",
+        "port": 1998,
+    },
+}
+
+
+# The core MFEs are added with a high priority, such that other users can override or
+# remove them.
+@MFE_APPS.add(priority=tutor_hooks.priorities.HIGH)
+def _add_core_mfe_apps(apps: dict[str, MFE_ATTRS_TYPE]) -> dict[str, MFE_ATTRS_TYPE]:
+    apps.update(CORE_MFE_APPS)
+    return apps
```

Comment:
Let's not add ecommerce MFE apps with a high priority. This should only be done for core MFE apps. Similarly, let's not create a CORE_MFE_APPS, which is an ambiguous name. Instead, I suggest:

```
@MFE_APPS.add()
def _add_ecommerce_mfe_apps(apps: dict[str, MFE_ATTRS_TYPE]) -> dict[str, MFE_ATTRS_TYPE]:
    apps.update({
        "orders": {
            "repository": "https://github.com/edx/frontend-app-ecommerce",
            "port": 1996,
        },
        "payment": {
            "repository": "https://github.com/edx/frontend-app-payment",
            "port": 1998,
        },
    })
    return apps
```